pkg/install/1-installresources.go

```diff
@@ -668,9 +651,20 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbClient := mgmtnetwork.NewLoadBalancersClient(i.env.SubscriptionID())
+			lbClient.Authorizer = i.fpAuthorizer
+			resourceGroupIDSplit := strings.Split(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, "/")
+			resourceGroupName := resourceGroupIDSplit[len(resourceGroupIDSplit)-1]
+			loadBalancer, err := lbClient.Get(ctx, resourceGroupName, "aro-internal-lb", "")
```

Comment:
Move this to the Installer Interface and make sure it is created when new Installer object is created